api/specs.proto

```diff
@@ -160,6 +160,11 @@ message ContainerSpec {
 	// StopGracePeriod the grace period for stopping the container before
 	// forcefully killing the container.
 	Duration stop_grace_period = 9;
+
+	// RegistryAuth is the registry auth token obtained from the client, required
+	// to pull private images
+	// TODO(nishanttotla): This field will later be deprecated
+	string registry_auth = 10;
```

Comment:
I thought of that, but figured it'll be easier (barely though) to remove when it's the last field with no renumbering needed. Can move if that's preferred.